src/saml2/response.py

```diff
@@ -216,16 +216,16 @@ def _verify(self):
                                                         self.request_id))
             return None
 
-        assert self.response.version == "2.0"
+        assert self.response.version == "2.0", "Bad response version: %s" % self.response.version
         if self.response.destination and \
             self.response.destination != self.return_addr:
             if self.log:
                 self.log.error("%s != %s" % (self.response.destination,
                                                 self.return_addr))
             return None
 
-        assert self.issue_instant_ok()
-        assert self.status_ok()
+        assert self.issue_instant_ok(), "failed issue_instant_ok"
```

Comment:
I suspect one of these assertions is failing for our customer and not being logged.